aws/alb.tf

```diff
@@ -27,12 +23,7 @@ resource "aws_security_group" "load_balancer_security_group" {
     to_port     = 443
     protocol    = "tcp"
     description = "dataeng team ips"
-    cidr_blocks = [
-      "50.168.68.90/32", # office
-      "76.140.96.213/32",
-      "67.2.180.194/32",
-      "73.63.25.63/32"
-    ]
+    cidr_blocks = var.whitelist_cidr_blocks
```

Comment:
Does this include the VPN? Or do I need to send my IP still?

Reply:
No it doesn't. I need to get help from the infra team to figure that out.